src/v2/controllers/FormController.ts

```diff
@@ -149,6 +151,11 @@ export default Controller.extend({
 
   // eslint-disable-next-line max-statements
   async handleInvokeAction(actionPath = '', actionParams = {}) {
+    if (actionPath === ORG_PASSWORD_RECOVERY_LINK) {
+      this.options.appState.trigger('restartLoginFlow', 'resetPassword');
```

Comment:
We need to start new reset password flow from scratch, because reusing state handle of started login flow can lead to issue, see video 1

Without this fix:

https://github.com/okta/okta-signin-widget/assets/72614880/279de18d-478c-42f2-ab3a-ff0ff12e6ffd

With the fix:

https://github.com/okta/okta-signin-widget/assets/72614880/c290f6af-e27f-4832-b868-1a5d4080dfea